### ISSUE TITLE
Revamping Buttons and Text

### ISSUE THREAD

**Issue by tanishq-arya** [khannakshat7/Elektra#19]
The intro text is small for the desktop view and I would like to change the button to a larger size.
![Screenshot (942)_LI](https://user-images.githubusercontent.com/77333275/110452621-5a902c80-80eb-11eb-8622-f408d62fd57d.jpg)

**Comment by deeppatel23** [comment]
As a GSSOC'21 Participant, I would like to work on this issue.

**Comment by khannakshat7** [comment]
@tanishq-arya Do you want to work on the issue?

**Comment by tanishq-arya** [comment]
Yes I would like to work on this. Thankyou for assigning me this issue.

**Comment by tanishq-arya** [comment]
![image](https://user-images.githubusercontent.com/77333275/110632321-7e2ba380-81cd-11eb-9d5e-642f0efd5a4e.png)

I have increased the size of text and changed the buttons. But I'm not sure where to put the CSS, because when I try to change the style.css the changes are not visible. 
Also another issue is that when we go in the mobile-nav on smaller screens on hover the navigation color turns red. I have tried but I cannot find how it is still red.

**Comment by tanishq-arya** [comment]
I have completed the styling and committed the changes.  I have one small problem I think I didn't open a new branch so now the commits have been made to the Contact-form UI issue pull request. Can you suggest me what should I do now ?

**Comment by tanishq-arya** [comment]
#17 This issue is solved in the PR mentioned
